Utils/Dataflow/069_upload2es/load_data.sh

```diff
@@ -10,6 +10,17 @@ $(basename "$0") [-c CONFIG] [FILE...]
 PARAMETERS:
   CONFIG -- configuration file
   FILE   -- file in NDJSON format for loading to Elasticsearch via bulk interface
+
+OPTIONS:
+  -b, --batch {e[abled]|d[isabled]} Specifies batch-mode: (e)nabled|(d)isabled.
```

Comment:
I think it could be handy to use this option as a flag: presented = enabled, missed = disabled.
Yet to make it possible to use it with value, we need a bit of invention in parsing:
```
BATCHMODE='disabled'
case $1 in
  ...
  -b|--batch)
    if [ -z "$2" ] || [[ "$2" == -* ]]; # it is last command line argument, or the next one starts with '-'
    then
      BATCHMODE='enabled'
    else
      BATCHMODE="$2"
      shift
    fi
    ;;
  ...
esac
```

Reply:
@mgolosova 
That looks very useful. I think, I should apply it right now. Thank you! :)